src/firebase/firebaseApp.ts

```diff
@@ -16,6 +21,11 @@ const firebaseConfig = {
 };
 
 // Initialize Firebase
-const app = initializeApp(firebaseConfig);
+const firebaseApp = firebase;
+
+if (firebaseConfig?.projectId) {
```

Comment:
what's the purpose of this code

Reply:
Why is it necessary to check for projectId? What is the case where `projectId` wouldn't exist that we need to account for?